OPHD/States/MapViewStateTurn.cpp

```diff
@@ -363,6 +363,7 @@ void MapViewState::updateBiowasteRecycling()
 
 			Residence* residence = static_cast<Residence*>(*residenceIterator);
 			residence->pullWaste(recycling->wasteProcessingCapacity());
+			++residenceIterator;
```

Comment:
Whoops, that was me. I introduced this bug during the attempt at refactoring in dff1b5e25286c8461e96cbfc8aac0b80d5ca3230 (PR #756). Thank you for catching and fixing it.

Reply:
I know, I was going to use git blame and then thought, nah, it's an easy mistake to make and I didn't catch it when reading the code, only when I noticed that biowaste wasn't being removed from residences even though I had a recycling facility. Looked at the code and noticed we never traversed the container.